### PR TITLE
issues-55 fixed

### PR DESCRIPTION
**Pull request by pthakkar02**
@magwas Kindly review the pull request.

### CODE REVIEW

**Comment by magwas** [comment]
You have PMD violations. You should see them in the problems tab in your eclipse. What is your discord username?

File: src/main/java/com/kodekonveyor/work_request/revoke/RevokeWorkRequestController.java [hunk 1]
```diff
@@ -18,13 +20,27 @@
...
     inputValidation(workRequestId);
 
     final WorkRequestEntity workRequestEntity =
         workRequestRepository.findByWorkRequestId(workRequestId).get(0);
-    workRequestRepository.delete(workRequestEntity);
+
+    final UserEntity user = authenticatedUserService.call();
+    final UserEntity customer = workRequestEntity.getCustomer();
+
+    //a user can only revoke its own work request
```
Comment (magwas) [inline src/main/java/com/kodekonveyor/work_request/revoke/RevokeWorkRequestController.java]: please do not use comments. See https://repository.kodekonveyor.com/metamodel/kode-konveyor/develop/codingrules.html The forth rule.

File: src/test/java/com/kodekonveyor/work_request/WorkRequestEntityTestData.java [hunk 2]
```diff
@@ -28,6 +30,16 @@ public static WorkRequestEntity get() {
     return workRequestEntity;
   }
 
+  public static WorkRequestEntity getRevoke() {
```
Comment (magwas) [inline src/test/java/com/kodekonveyor/work_request/WorkRequestEntityTestData.java]: The name of the getter should reflect the most important field and how it is set up in the Testdata.
The getters should call get()  as the first call, and just modify what is needed to be modified.
/metamodel/kode-konveyor/develop/codingrules.html#9ec42823-7056-4e73-b1f4-600a066c8d88

Anyway, in your case it seems you can do the test with the default work request, no need to create a new one.

File: src/main/java/com/kodekonveyor/work_request/revoke/RevokeWorkRequestController.java (hunk 1 above)
Comment (pthakkar02) [reply src/main/java/com/kodekonveyor/work_request/revoke/RevokeWorkRequestController.java]: @magwas, Fixed this change.

File: src/test/java/com/kodekonveyor/work_request/WorkRequestEntityTestData.java (hunk 2 above)
Comment (pthakkar02) [reply src/test/java/com/kodekonveyor/work_request/WorkRequestEntityTestData.java]: @magwas, Fixed this change.

**Pull request merged**